helm/fcp-fd-probe/templates/azure-identity-binding.yaml

```diff
@@ -0,0 +1,3 @@
+{{- include "ffc-helm-library.azure-identity-binding" (list . "fcp-fd-auth.azure-identity-binding") -}}
```

Comment:
Should reference the repo name so `fcp-fd-probe.azure-identity-binding` instead of `fcp-fd-auth.azure-identity-binding`.